### PR TITLE
Add details for releasing to Hackage

### PR DESCRIPTION
**Pull request by pbrisbin**

### CODE REVIEW

File: haskell-open-source.md [hunk 1]
```diff
@@ -64,3 +64,27 @@ To determine what changes are "breaking", use the following:
...
+
+## Release
+
+All (interested) Frecklers should be in the Maintainers group of our Hackage
+libraries, with permission to upload new versions.
+
+To release a new version:
+
+1. Update the package version
+1. Add a `CHANGELOG` entry
```
Comment (cbeav) [inline haskell-open-source.md:76]: 2. 3. 4. ?

Comment (cbeav) [reply haskell-open-source.md:76]: Wtf is that styling GH, Jesus. Was asking if you meant to enumerate beyond the number 1?

Comment (cbeav) [inline haskell-open-source.md:76]: Enumerate beyond 1?

**Review by cbeav — APPROVED** [review]

File: haskell-open-source.md (hunk 1 above)
Comment (jdreaver) [reply haskell-open-source.md:76]: Bruh lrn2markdown :trollface: 

(Markdown automatically enumerates these for you when it renders.) https://github.com/freckle/guides/blob/dc92bdabeea48476df79f4bd47c076e94f954051/haskell-open-source.md

Comment (pbrisbin) [reply haskell-open-source.md:76]: Wait, do you not know about the feature of Markdown where you don't need to actually number lists?

Comment (cbeav) [reply haskell-open-source.md:76]: Wow feelin' pretty Markdown-shamed right now. Can't believe I've been typing out numbers all these years like some plebe. At least give me the moral victory of admitting this is some wack-looking syntax.

Comment (pbrisbin) [reply haskell-open-source.md:76]: https://xkcd.com/1053/

Comment (eborden) [reply haskell-open-source.md:76]: This is my "least-favorite favorite" feature of markdown. I was quite confused when I saw it for the first time, said "oh that's handy", and I'm still confused to this day.

File: haskell-open-source.md [hunk 2]
```diff
@@ -64,3 +64,27 @@ To determine what changes are "breaking", use the following:
 ![](./pvp-chart.png)
 
 With `A=EPOCH`, `B=MAJOR`, and `C=MINOR`.
+
+## Release
+
+All (interested) Frecklers should be in the Maintainers group of our Hackage
```
Comment (eborden) [inline haskell-open-source.md:70]: Opens issue against hackage asking for a maintainer groups feature.

File: haskell-open-source.md [hunk 3]
```diff
@@ -64,3 +64,27 @@ To determine what changes are "breaking", use the following:
...
+
+To release a new version:
+
+1. Update the package version
+1. Add a `CHANGELOG` entry
+1. Open the above in a Pull Request and get review
+1. Once merged, run the following from `main`:
+
+   ```console
+   stack upload --pvp-bounds both .
```
Comment (eborden) [inline haskell-open-source.md:81]: I'm not a huge fan of `--pvp-bounds`, but I also don't have a better answer and don't want people to waste time on bounds, so... :shrug:

**Review by eborden — APPROVED** [review]

File: haskell-open-source.md (hunk 3 above)
Comment (pbrisbin) [reply haskell-open-source.md:81]: Yeah, big ditto there.

**Pull request merged**